docs/pt/docs/advanced/templates.md

```diff
@@ -0,0 +1,119 @@
+# Templates
+
+Você pode usar qualquer template engine com o **FastAPI**.
+
+Uma escolha comum é o Jinja2, o mesmo usado pelo Flask e outras ferramentas.
+
+Existem utilitários para configurá-lo facilmente que você pode usar diretamente em sua aplicação **FastAPI** (fornecidos pelo Starlette).
+
+## Instalar dependencias
+
+Para instalar o `jinja2`, siga o código abaixo:
+
+<div class="termy">
+
+```console
+$ pip install jinja2
+```
+
+</div>
+
+## Usando `Jinja2Templates`
+
+* Importe `Jinja2Templates`.
+* Crie um `templates` que você pode reutilizar posteriormente.
+* Declare um parâmetro `Request` no *path operation* que retornará um template.
+* Use os `templates` que você criou para renderizar e retornar uma `TemplateResponse`, passando o nome do template, o o request object, e um "context" dict com pares key-value a serem usados dentro do template Jinja2.
+
+```Python hl_lines="4  11  15-18"
+{!../../../docs_src/templates/tutorial001.py!}
+```
+
+!!! Nota
+    Antes do FastAPI 0.108.0, Starlette 0.29.0, `name` era o primeiro parâmetro.
+
+    Além disso, em versões anteriores, o objeto `request` era passado como parte dos pares key-value no contexto para Jinja2.
+
+
+!!! Dica
+    Ao declarar `response_class=HTMLResponse`, a interface dos documentos será capaz de saber que a resposta será HTML.
```

Comment:
2. Change `interface dos documentos` to `documentação`.
3. Change `será capaz de saber` to `entenderá`.